bot.py

```diff
@@ -273,6 +327,50 @@ async def add_field(self, interaction: discord.Interaction) -> None:
             )
 
 
+class FieldToRemove(discord.ui.View):
+    """
+    Subclass of the `discord.ui.View` class. Used for creating a select prompt.
+
+    Args:
+        placeholder (str): The placeholder text that will be displayed
+        in the channel select menu.
+        options (List[SelectOption]): A list of `SelectOption` instances that
+        will be displayed as options in the select prompt.
+        max_values (int, optional): The maximum number of options
+        that can be selected by the user. Default is 1.
+        ephemeral (bool, optional): A boolean indicating whether
+        the select prompt will be sent as an ephemeral message or not. Default is False.
+    """
+
+    def __init__(
+        self,
+        placeholder: str,
```

Comment:
Czy opis argumentów konstruktora nie powinien się znajdować w ciele konstruktora, a nie ciele klasy?

Reply:
To skomplikowane. Na podstawie mojej wiedzy, opis argumentów powinien znajdować się w ciele klasy. Odnośnie docstringów konstruktorów, z jednej strony znalazłem informacje, że powinny je również posiadać, z drugiej Pylint informuje mnie jedynie o braku docstringów ciał klas i ich metod, sugerując, że konstruktor nie potrzebuje takowego.

Odnośnik do tematu na stackoverflow:
https://stackoverflow.com/questions/62444612/should-i-list-class-methods-in-the-class-docstring

Przykład docstringów w stylu Google, z którego korzystam:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html